packages/turf-geojson-rbush/tsconfig.testTypes.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.shared.json",
+  "compilerOptions": {
+    "noEmit": true
+  },
+  "files": ["types.ts"]
```

Comment:
I'm actually a little surprised this doesn't need to declare `index.js`, I must be missing something about how the paths directive works or something.

Reply:
I don't think tsc can type check the JS implementation at all. So for the JS projects I believe it uses types.ts as an entry point, loads up index.d.ts (infers that's what you really mean when asking to import from index.js), and then checks what it can.

Reply:
Right but shouldn't `index.d.ts` have to be listed in files in that case?

Reply:
`files` seems more to be the top level entry points where you want tsc to start compiling. From there it uses whatever the module resolution policy is to find all the dependencies.

Reply:
https://www.typescriptlang.org/tsconfig/#files I guess it doesn't say that it has to be exhaustive, but I always read that as needing to list every file. 🤷